readme.md

```diff
@@ -1,20 +1,24 @@
 # Search bar for Airbnb
+
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+
+- [DEMO LINK](https://VasylKurtiak.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://VasylKurtiak.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
please add links to Pull Request description

Reply:
Thanks for the tips) I will fix it now.